frontend/src/metabase/public/components/widgets/SharingPane/InteractiveEmbeddingCTA/InteractiveEmbeddingCTA.tsx

```diff
@@ -0,0 +1,77 @@
+import { t } from "ttag";
+import Link from "metabase/core/components/Link";
+import {
+  ClickIcon,
+  CTAContainer,
+  CTAHeader,
+  ProBadge,
+  CTALinkText,
+} from "metabase/public/components/widgets/SharingPane/InteractiveEmbeddingCTA/InteractiveEmbeddingCTA.styled";
+import { getIsPaidPlan } from "metabase/selectors/settings";
+import { Text, Group, Stack } from "metabase/ui";
+import { useSelector } from "metabase/lib/redux";
+
+const getText = (isPaidPlan: boolean) => {
+  if (isPaidPlan) {
+    return {
+      showProBadge: false,
+      description: t`Your plan allows you to use Interactive Embedding create interactive embedding experiences with drill-through and more.`,
+      linkText: t`Set it up`,
+      linkTarget: "/admin/settings/embedding-in-other-applications/full-app",
+    };
+  }
+
+  return {
+    showProBadge: true,
+    description: t`Give your customers the full power of Metabase in your own app, with SSO, advanced permissions, customization, and more.`,
+    linkText: t`Learn more`,
+    linkTarget: "https://www.metabase.com/product/embedded-analytics",
+  };
+};
+
+export const InteractiveEmbeddingCTA = () => {
+  const isPaidPlan = useSelector(getIsPaidPlan);
+
+  const { showProBadge, description, linkText, linkTarget } =
+    getText(isPaidPlan);
+
+  const badge = (
+    // TODO: Check padding because design keeps using non-mantine-standard units
```

Comment:
Just need to make sure on the spacing, maybe there's another component we can use that has these paddings. I'd use `Badge` but it doesn't seem to be ready yet